apps/ui/src/components/inputs/editTextInput.tsx

```diff
@@ -0,0 +1,189 @@
+import type { Metadata } from '@bako-id/sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  type InputProps,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { Dialog } from '../dialog';
+import { CloseIcon } from '../icons/closeIcon';
+import { TrashIcon } from '../icons/trashIcon';
+
+interface IEditTextValueInput extends InputProps {
+  title: string;
+  modalType: string;
+  onSave: (metadata: Metadata) => Promise<void>;
```

Comment:
```suggestion
  onChange: (metadata: Metadata) => Promise<void>;
```

Reply:
Wasn't able to change the key from `onSave` to `onChange`, because the component extends `InputProps` and it already have declarated.

Reply:
```ts
interface IEditTextValueInput extends Omit<InputProps, 'onChange'> {
  title: string;
  modalType: string;
  onSave: (metadata: Metadata) => Promise<void>;
```